games-arcade/orthorobot/orthorobot-0-r2.ebuild

```diff
@@ -0,0 +1,41 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit eutils
+
+DESCRIPTION="Perspective based puzzle game, where you flatten the view to move across gaps"
+HOMEPAGE="http://stabyourself.net/orthorobot/"
+SRC_URI="http://stabyourself.net/dl.php?file=${PN}/${PN}-source.zip -> ${P}.zip"
+
+LICENSE="CC-BY-NC-ND-3.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+LVSLOT="0.8"
+RDEPEND=">=games-engines/love-0.8.0:${LVSLOT}"
```

Comment:
To be honest, I don't think the `>=` dep makes sense given that the slot can't contain an older version, by definition.